scripts/match.gd

```diff
@@ -178,5 +189,14 @@ func set_server_to(server):
 		serve_marker[0].hide()
 		serve_marker[1].show()
 
+## Play a sound by key, first checking if there is an existing wav to use,
+## and if not then attempting to use TTS as a fallback.
+func play_sound(key:String) -> void:
+	if sounds_dict.has(key) and sounds_dict[key] is AudioStreamWAV:
+		$AudioStreamPlayer.stream = sounds_dict[key]
+		$AudioStreamPlayer.play()
+	else:
+		Global.speak_tts(key)
```

Comment:
this function is what plays sounds during a match.

if called on one of the sound key strings that are defined in `globals.gd`, then we either play an asset if we have one, or we use TTS if not